### PR TITLE
Update text

### PR DESCRIPTION
**Pull request by lennymd**
Review text to fix any typos and clarify explanations.

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -3,143 +3,153 @@ layout: default
 title: api
 nav_order: 5
 ---
+
```
Comment (lorenzoromagnoli) [inline docs/api.md:6]: This file should not be edited manually as it is automatically generated based on the documentation written in the library js file using jsdoc-to-markdown.

Please document functions here https://github.com/id-studiolab/MLTK01/blob/master/library/mltk.js and regenerate the api.md by executing gulp generate-doc

**Review by lorenzoromagnoli — APPROVED** [review]
It is mostly typos and linting so I'd say it can be merged... Unfortunately the edits you did on the api documentation might be get completely ignored once we regenerate automatically the api documentation using the automated script

File: docs/api.md (hunk 1 above)
Comment (lennymd) [reply docs/api.md:6]: Okok; I think VSCode linted the page when I opened it. I'll use the gulp command to regenerate

File: docs/Gemfile [hunk 2]
```diff
@@ -15,3 +15,7 @@ group :jekyll_plugins do
   gem "jekyll-seo-tag"
   gem "just-the-docs"
 end
+
+gem "webrick", "~> 1.8"
```
[the remark this answers is not in the document]
Comment (lennymd) [reply docs/Gemfile:19]: Yes

File: arduino_sketches/MLTK01Servo/MLTK01Servo.ino [hunk 3]
```diff
@@ -479,42 +482,42 @@ void onLedRingCharacteristicWrite(BLEDevice central, BLECharacteristic character
     r = ledRing1Characteristic[0];
     g = ledRing1Characteristic[1];
     b = ledRing1Characteristic[2];
-    pixels.setPixelColor(0, pixels.Color(r, g, b));         // Moderately bright green color.
```
[the remark this answers is not in the document]
Comment (lennymd) [reply arduino_sketches/MLTK01Servo/MLTK01Servo.ino:482]: I'll clean it up, yes.

File: docs/api.md (hunk 1 above)
Comment (lennymd) [reply docs/api.md:6]: Screenshot 2023-02-16 at 11 12 19

So I'm awful at NPM/Node things right now. I installed gulp and tried running the command and got the following error.

Comment (lorenzoromagnoli) [reply docs/api.md:6]: I think there is a bug in the library that keeps giving me this error as well... the only workaround I found so far is to enter the gulp file here https://github.com/id-studiolab/MLTK01/blob/0f42b1938f168c3e84a4fcc125f1ac02f833f199/gulpfile.js#L12-L13

- Uncomment one line 12 and comment line 13, save the file and run the gulp generate doc command. 
- Then do the opposite, resave and run the command again.

It seems like somehow the template file get cached or something and the second time you run it with the same content doesn't work

Comment (lorenzoromagnoli) [reply docs/api.md:6]: actually, fuck gulp! 
I just pushed some modification to master to be able to generate the documentation with a npm task... if you pull master you should be able to run npm run docs and the api documentation should automatically update.

**Pull request merged**